README.adoc

```diff
@@ -1,88 +1,323 @@
+
+The Maven plugin that is used to build and deploy the quickstarts can also generate fully rendered `README.html` instructions from the `README.adoc` files.
+
+To build all of the quickstart `README.html` files, including the root `README.html` file that contains the table with links to all available quickstarts, navigate to the root folder of the quickstarts and run the same command.
+
+[source,options="nowrap"]
+----
+$ mvn clean generate-resources -Pdocs
+----
+
+If you see errors about missing dependencies, verify that you xref:build_wildfly_server_and_boms[built the version of the WildFly server and BOMs] that corresponds to the quickstart repository you are using.
+
+[NOTE]
+====
+To build the `README.html` file for a specific quickstart, navigate to the quickstart directory and run the following command.
 
-All you need to build these projects is {buildRequirements}. See link:{configureMavenDocUrl}[Configure Maven to Build and Deploy the Quickstarts] to make sure you are configured correctly for testing the quickstarts.
+[source,options="nowrap"]
+----
+$ mvn clean generate-resources -Pdocs
+----
+====
 
-To run these quickstarts with the provided build scripts, you need the {productName} distribution ZIP. For information on how to install and run JBoss, see the {LinkGettingStartedGuide}[__{GettingStartedBookName}__] located on the Red Hat Customer Portal.
 
-You can also run the quickstarts in xref:run_the_quickstarts_in_jboss_developer_studio_or_eclipse[{JBDSProductName} or Eclipse].
+[[suggested_approach_to_the_quickstarts]]
+== Suggested Approach to the Quickstarts
+
+We suggest you approach the quickstarts as follows:
+
+* Regardless of your level of expertise, we suggest you start with the `helloworld` quickstart. It is the simplest example and is an easy way to prove the server is configured and running correctly.
+* If you are a beginner or new to JBoss, start with the quickstarts labeled `Beginner`, then try those marked as `Intermediate`. When you are comfortable with those, move on to the `Advanced` quickstarts.
+* Some quickstarts are based upon other quickstarts but have expanded capabilities and functionality. If a prerequisite quickstart is listed, make sure you deploy and test it before looking at the expanded version.
+
 
 [[run_the_quickstarts]]
-== Run the Quickstarts
+== Run the Quickstarts Using the Maven Command Line
 
 [[build_and_deploy_a_quickstart]]
 === Build and Deploy a Quickstart
 
-The root folder of each individual quickstart contains a `README{outfilesuffix}` file with detailed instructions on how to build and run the example. In most cases you do the following:
```

Comment:
I think the old content here was fine, user should be redirected to specific QS docs.

Reply:
The problem is that attributes are not substituted when you read this in GitHub, so I replaced all attributes in the WildFly version of the README.adoc file with actual values.